coremlpython/CoreMLPythonUtils.mm

```diff
@@ -4,6 +4,15 @@
 #include <pybind11/eval.h>
 #include <pybind11/numpy.h>
 
+#include <numpy/arrayobject.h>
+
+#if PY_MAJOR_VERSION < 3
+#define PyBytes_Check(name) PyString_Check(name)
+#define PyAnyInteger_Check(name) (PyLong_Check(name) || PyInt_Check(name))
+#else
+#define PyAnyInteger_Check(name) (PyLong_Check(name) || (_import_array(), PyArray_IsScalar(name, Integer)))
```

Comment:
There is probably a better location where I can place `_import_array()`, but I wasn't sure of an appropriate entry point. This should be very fast after the first import though.